modules/core/src/alloc.cpp

```diff
@@ -81,6 +82,10 @@ cv::utils::AllocatorStatisticsInterface& getAllocatorStatistics()
     return allocator_stats;
 }
 
+#if defined HAVE_POSIX_MEMALIGN || defined HAVE_MEMALIGN
+static const bool useMemalign = cv::utils::getConfigurationParameterBool("OPENCV_ENABLE_MEMALIGN", false);
```

Comment:
We should avoid "initialization order fiasco".

Variable can be used below without proper initialization and then crash in mismatched `fastFree()` calls.